pkg/provider/kubernetes/crd/traefikio/v1alpha1/middleware.go

```diff
@@ -67,6 +67,7 @@ type ErrorPage struct {
 	// as multiple comma-separated numbers (500,502),
 	// as ranges by separating two codes with a dash (500-599),
 	// or a combination of the two (404,418,500-599).
+	// +kubebuilder:validation:items:Pattern=`^[-0-9,]+$`
```

Comment:
This regex doesn't validate much, as for instance `-,` would be a valid input.
I'm not sure this is really needed though.